ParisBNF/abb/BNFabb92.xml

```diff
@@ -0,0 +1,682 @@
+                                    ወጸውዖሙ፡ ወይቤ፡ ሎሙ፡ ወአዘዞሙ፡ ይስግዱ፡ ለጣዖት። ወአበዩ፡ ሰጊደ፡ ለጣዖት፡ ወፈቀደ፡ ንጉሥ፡
+                                    ይሖር፡ ፀብአ </incipit>
+                                <note>In previous catalogues this miracle was not distinguished from
+                                    the preceding vita.</note>
+                            </msItem>
+
+                            <msItem xml:id="ms_i18">
+                                <locus from="163ra" to="168vb"/>
+                                <title type="complete" ref="LIT6173JamesIn"/>
+                                <note>read on <date ref="ethiocal:Hedar27">27 Ḫǝdār</date>
+                                </note>
+                                <incipit xml:lang="gez">ስምዑ፡ ለቅዱስ፡ ያ<hi rend="rubric">ዕቆብ፡ ግሙድ፡
+                                        መስ</hi>ተጋድል፡ ዘረከቦ፡ ዘ<hi rend="rubric">ተፈጸመ፡ ስምዑ፡ በ</hi>ምድረ፡
+                                    ፋርስ፡ በሀገር፡ እንተ፡ ስማ፡ በልቃ፡ በምንግሥተ፡ ፋርስ፡ በመዋዕለ፡ ሰርዳ፡ ንጉሥ፡ ወውእቱ፡
+                                    ክርስቲያናዊ፡ የአምን፡ በክርስቶስ፡ በሰላመ፡ እግዚአብሔር፡ አሜን።</incipit>
+                            </msItem>
+                        </msItem>
+                    </msContents>
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf" quantity="168">168</measure>
+                                    <dimensions type="outer">
+                                        <height unit="mm">330</height>
+                                        <width unit="mm">235</width>
+                                    </dimensions>
+                                </extent>
+                                <foliation>Foliation in the upper right corner of the
+                                    folia.</foliation>
+                                <collation>
+                                    <note>Due to the nature of modern library rebinding, the quire
+                                        quire structure is lost.</note>
+                                </collation>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout columns="2" writtenLines="30 33">
+                                    <ab type="pricking">Pricking is hardly visible.</ab>
+                                    <ab type="ruling">Ruling is visible.</ab>
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc>
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <seg type="script">15/16th-century script</seg>
```

Comment:
a date element can be added to handNote

Reply:
I would actually not use script unless referring to for example Gwelh script. I would have something like:
```
   <seg type="ink">Black, red.</seg>
<date notBefore="1400" notAfter="1599"/>
<desc>Careful, regular; by a good scribe. <persName
                                        ref="PRS1071dAbbadi"/> notes the presence of <cit>
                                        <quote>arabesques and hieroglyphs</quote>
                                    </cit> (<cit>
                                        <quote xml:lang="fr">arabesques et hiéroglyphes</quote>
                                    </cit>) .
</desc>
```

Reply:
Thank you!